src/services/apiService.js

```diff
@@ -72,6 +72,16 @@ export const startQuizSession = (quizId) =>
       console.error(`Error starting quiz: ${e}`);
     });
 
+export const fetchRecordings = () =>
+  api.get(`https://api.edukona.com/recordings/`).catch((error) => {
```

Comment:
Let's omit `https://api.edukona.com` and have it captured from baseURL at the top; this will be necessary to do before we start using release branches and pre-prod environments.